themes/wilson_theme_starterkit/components/atoms/button/button.stories.js

```diff
@@ -0,0 +1,41 @@
+export default { title: 'Buttons' };
+
+import button from './button.twig';
+import '../../../css/styles.css'
+import drupalAttribute from 'drupal-attribute'
+
+export const default_button = () => (
```

Comment:
Does there have to be something called 'default' or would it be clearer to just call this 'primary_button' so it ties in with the type?

Reply:
Nope, changed this to default which is a bit more in-keeping 👍